src/styles/more-light/theme.pcss

```diff
@@ -208,6 +208,8 @@
   --red-700:#804646;
   --red-800:#552f2f;
   --red-900:#2b1717;
+
+  --button-size: 2.357rem;
```

Comment:
I'm anxious, that this declaration has side effects to some buttons. Have you checked all main buttons?